README.md

```diff
@@ -1,26 +1,29 @@
-## The Golden Rule:
+# To Do App
 
-🦸 🦸‍♂️ `Stop starting and start finishing.` 🏁
+## Database
 
-If you work on more than one feature at a time, you are guaranteed to multiply your bugs and your anxiety.
+-   Set up a table with
+    -   id
+    -   created_at (date)
+    -   description (varchar)
+    -   complete (bool)
+    -   user_id (foreign key) - default value
+-   Set up our policies - ensure that user_id on the Item matches the authed user
+-   Add a couple of rows
 
-## Making a plan
+## Items Page
 
-1. **Make a drawing of your app. Simple "wireframes"**
-1. **Once you have a drawing, name the HTML elements you'll need to realize your vision**
-1. **For each HTML element ask: Why do I need this?**
-1. **Once we know _why_ we need each element, think about how to implement the "Why" as a "How"**
-1. **Find all the 'events' (user clicks, form submit, on load etc) in your app. Ask one by one, "What happens when" for each of these events. Does any state change?**
-1. **Think about how to validate each of your features according to a Definition of Done**
-1. **Consider what features _depend_ on what other features. Use this dependency logic to figure out what order to complete tasks.**
-
-Additional considerations:
-
--   Ask: which of your HTML elements need to be hard coded, and which need to be dynamically generated?
--   Consider your data model.
-    -   What kinds of objects (i.e., Dogs, Friends, Todos, etc) will you need?
-    -   What are the key/value pairs?
-    -   What arrays might you need?
-    -   What needs to live in a persistence layer?
--   Is there some state we need to initialize?
--   Ask: should any of this work be abstracted into functions? (i.e., is the work complicated? can it be resused?)
+1. Displaying the list of Items
```

Comment:
Your planning here is brilliant. Love how you added in the steps needed to complete this lab and even thought through the applicable functions. Great work!